terraform/environments/cdpt-chaps/database.tf

```diff
@@ -0,0 +1,69 @@
+#-----------------------------------------------------------------------------
```

Comment:
Some of the spacing looks a bit funny in places. There are also a couple of files missing end of file characters.